src/PrestaShopBundle/Form/Admin/Sell/Address/CustomerAddressType.php

```diff
@@ -0,0 +1,448 @@
+                    new NotBlank([
+                        'message' => $this->translator->trans(
+                            'This field cannot be empty', [], 'Admin.Notifications.Error'
+                        ),
+                    ]),
+                ],
+            ])
+            ->add('phone', TextType::class, [
+                'required' => $this->isRequired(RequiredFields::REQUIRED_FIELD_PHONE, $requiredFields),
+                'constraints' => [
+                    $this->getNotBlackOrNull(RequiredFields::REQUIRED_FIELD_PHONE, $requiredFields),
+                    new CleanHtml(),
+                    new TypedRegex([
+                        'type' => 'phone_number',
+                    ]),
+                    new Length(
+                        [
+                            'max' => AddressConstraint::MAX_PHONE_LENGTH,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters',
+                                ['%limit%' => AddressConstraint::MAX_PHONE_LENGTH],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]
+                    ),
+                ],
+            ])
+            ->add('other', TextareaType::class, [
+                'required' => $this->isRequired(RequiredFields::REQUIRED_FIELD_OTHER, $requiredFields),
+                'constraints' => [
+                    $this->getNotBlackOrNull(RequiredFields::REQUIRED_FIELD_OTHER, $requiredFields),
+                    new CleanHtml(),
+                    new TypedRegex([
+                        'type' => 'message',
+                    ]),
+                    new Length(
+                        [
+                            'max' => AddressConstraint::MAX_OTHER_LENGTH,
+                            'maxMessage' => $this->translator->trans(
+                                'This field cannot be longer than %limit% characters',
+                                ['%limit%' => AddressConstraint::MAX_OTHER_LENGTH],
+                                'Admin.Notifications.Error'
+                            ),
+                        ]
+                    ),
+                ],
+            ]);
+
+        $builder->addEventSubscriber(new CustomerAddressFormSubscriber($this->stateChoiceProvider));
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults(
+            [
+                'constraints' => [
+                    new CustomerAddressZipCode(),
+                    new CustomerAddressCountryRequiredFields(),
+                ],
+            ]
+        );
+    }
+
+    /**
+     * @param string $field
+     * @param array $requiredFields
+     *
+     * @return NotBlank|null
+     */
+    private function getNotBlackOrNull(string $field, array $requiredFields): ?NotBlank
```

Comment:
```suggestion
    private function getNotBlanckOrNull(string $field, array $requiredFields): ?NotBlank
```

Reply:
`getNotBlankOrNull` is the correct wording :slightly_smiling_face:

Reply:
Indeed 😅